registry/registry_test.go

```diff
@@ -14,6 +14,12 @@ import (
 	"github.com/docker/distribution/registry/client/transport"
 	"github.com/docker/docker/api/types"
 	registrytypes "github.com/docker/docker/api/types/registry"
+
+	"github.com/stretchr/testify/require"
+	"github.com/docker/go-connections/tlsconfig"
+
+	"path/filepath"
+	"os"
```

Comment:
Use proper ordering for imports and always run `gofmt` on your code before making any PR.